features/git-town-kill/supplied_branch/feature_branch/with_tracking_branch.feature

```diff
@@ -21,8 +21,11 @@ Feature: git town-kill: killing the given feature branch
     Then it runs the commands
       | BRANCH       | COMMAND                       |
       | good-feature | git fetch --prune             |
+      |              | git add -A                    |
+      |              | git stash                     |
```

Comment:
The changes get stashed only when you supply a branch name other than the current branch.